pkg/controllers/update.go

```diff
@@ -77,6 +77,11 @@ func RunInstallScript() (bool, string, Error) {
 			message = "Error: update failed due to improper permissions\nPlease re-run with `sudo` or as an admin"
 		}
 
+		gnupgError := exitCode == 3
+		if gnupgError {
+			message = "Error: Unable to find gpg binary for signature verification\nYou can resolve this error by installing your system's gnupg package"
+		}
```

Comment:
Can you make this an `else if` off of the above conditional? Just so we're certain that only one branch runs. Something like:

```
permissionError := exitCode == 2 || ...
gnupgError := exitCode == 3
if permissionError {
  ...
} else if gnupgError {
 ...
}
```